tests/nested/manual/core20-factory-reset-install-device-hook/pc-snap-decl-extras.json

```diff
@@ -0,0 +1,9 @@
+{
+    "plugs": {
+        "system-files": {
+            "allow-installation": "true",
+            "allow-auto-connection": "true"
+        }
+    },
+    "format": "1"
+}
```

Comment:
missing newline

Reply:
Fixed, also fixed original file that I have reused